[2023-06-30T13:06:37.038+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: order-letter_NER.post_to_dynamics_task manual__2023-06-30T12:49:03.146226+00:00 [queued]>
[2023-06-30T13:06:37.087+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: order-letter_NER.post_to_dynamics_task manual__2023-06-30T12:49:03.146226+00:00 [queued]>
[2023-06-30T13:06:37.098+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 6
[2023-06-30T13:06:37.220+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): post_to_dynamics_task> on 2023-06-30 12:49:03.146226+00:00
[2023-06-30T13:06:37.265+0000] {standard_task_runner.py:57} INFO - Started process 17920 to run task
[2023-06-30T13:06:37.284+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'order-letter_NER', 'post_to_dynamics_task', 'manual__2023-06-30T12:49:03.146226+00:00', '--job-id', '107', '--raw', '--subdir', 'DAGS_FOLDER/order-letter-extraction.py', '--cfg-path', '/tmp/tmpu99b14pj']
[2023-06-30T13:06:37.316+0000] {standard_task_runner.py:85} INFO - Job 107: Subtask post_to_dynamics_task
[2023-06-30T13:06:37.878+0000] {task_command.py:410} INFO - Running <TaskInstance: order-letter_NER.post_to_dynamics_task manual__2023-06-30T12:49:03.146226+00:00 [running]> on host f7b244b51c6e
[2023-06-30T13:06:38.431+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Abin Jilson' AIRFLOW_CTX_DAG_ID='order-letter_NER' AIRFLOW_CTX_TASK_ID='post_to_dynamics_task' AIRFLOW_CTX_EXECUTION_DATE='2023-06-30T12:49:03.146226+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-30T12:49:03.146226+00:00'
[2023-06-30T13:06:38.438+0000] {logging_mixin.py:149} INFO - 'AMSTERDAM Lamp', 'TOKYO Guest Chair,\r\nblue', 'ROME Guest Chair, green', 'Conference Bundle 1-6', 'ANTWERP\r\nConference Table.'
[2023-06-30T13:06:38.438+0000] {logging_mixin.py:149} INFO - 11, 6, 13, 2, 2
[2023-06-30T13:06:40.073+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/order-letter-extraction.py", line 176, in dynamics_upload_main
    }
IndexError: string index out of range
[2023-06-30T13:06:40.097+0000] {taskinstance.py:1345} INFO - Marking task as UP_FOR_RETRY. dag_id=order-letter_NER, task_id=post_to_dynamics_task, execution_date=20230630T124903, start_date=20230630T130637, end_date=20230630T130640
[2023-06-30T13:06:40.118+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 107 for task post_to_dynamics_task (string index out of range; 17920)
[2023-06-30T13:06:40.156+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-06-30T13:06:40.182+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
